packages/client/test/benchmarks/SigningUtil.test.ts

```diff
@@ -28,13 +28,15 @@ describe('SigningUtil', () => {
     describe.each(PAYLOAD_SIZES)('payload size: %s', (payloadSize: number) => {
 
         let wallet: Wallet
-        let signature: string
+        let hexSignature: string
+        let binarySignature: Uint8Array
         let payload: string
 
         beforeEach(async () => {
             wallet = fastWallet()
             payload = randomString(payloadSize)
-            signature = await wallet.signMessage(payload)
+            hexSignature = await wallet.signMessage(payload)
+            binarySignature = hexToBinary(hexSignature)  
```

Comment:
We should maybe benchmark two equal functionalities, i.e. two method which produce binary signature from a given output. That way the results are comparable.